build/media-suite_config.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# shellcheck disable=SC2034,SC1090,SC1117,SC1091,SC2119
+shopt -s extglob
+
+if [[ x"$LOCALBUILDDIR" = "x" ]]; then
+    printf '%s\n' \
+        "Something went wrong." \
+        "MSYSTEM: $MSYSTEM" \
+        "pwd: $(cygpath -w "$(pwd)")" \
+        "fstab: " \
+        "$(cat /etc/fstab)" \
+        "Create a new issue and upload all logs you can find, especially compile.log"
+    read -r -p "Enter to continue" ret
+    exit 1
+fi
+
+while true; do
+  case $1 in
+    --ffmpegVersion=* ) ffmpegVersion="${1#*=}"; shift ;;
+    -- ) shift; break ;;
+    -* ) echo "Error, unknown option: '$1'."; exit 1 ;;
+    * ) break ;;
+  esac
+done
+
+[ "$ffmpegVersion" == '' ] && ffmpegVersion='latest'
+sed -ri '/readme.txt/{s/(ffmpeg-).*(-win)/\1'$ffmpegVersion'\2/g}' "$LOCALBUILDDIR"/media-suite_helper.sh
```

Comment:
No.
Why would you need to change suite files for this? This would break git-synced suite installations.

Reply:
Is there a script for the suite to update?

Reply:
Yes, and there's the fact that it's git.